wordpress-mobile/gutenberg-mobile
# iOS: Fixes frozen editor scrolling in landscape on iPhone

Fixes: https://github.com/wordpress-mobile/WordPress-iOS/issues/16476

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/31795
`WordPress-iOS` PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/16497

To test:
See steps at https://github.com/WordPress/gutenberg/pull/31795

PR submission checklist:
